src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.spec.ts

```diff
@@ -170,14 +173,56 @@ describe('OsdListComponent', () => {
     });
   });
 
+  describe('show osd actions as defined', () => {
```

Comment:
Can we please name this `show action buttons and drop down actions depending on permissions` to make it clear, that these is just another duplicated block like the other 8 occurrences of the same text mentioned above? This would make the refactoring easier!